specification/implementation/metadata/mets/filesec/index.md

```diff
@@ -1,8 +1,10 @@
 ### 5.3.5.	Use of the METS file section (element fileSec)
-Use of the METS fileSec element is highly recommended by the CSIP (although not mandatory). It should describe all components of the IP which have not been already included in the amdSec and dmdSec elements. For all files the location and checksum need to be available. The METS file section serves as a manifest, allowing users to check that all files are present and a package is complete as well as testing the integrity of package files using checksum values.
+Use of the METS fileSec element is highly recommended by the CSIP (although not mandatory). It should describe all components of the IP which have not been already included in the amdSec and dmdSec elements. For all files the location and checksum need to be available.
 
 The METS file section serves as a manifest, allowing users to check that all files are present and a package is complete as well as testing the integrity of package files using checksum values.
 
+The files are described in an element named file group `fileGrp`. The `fileGrp` element contains the descriptions of the files including their checksum and location. It is possible have `fileGrp` elements inside `fileGrp` elements, something this specification don't encourages instead there SHOULD not occur any nesting of `fileGrp`elements.
```

Comment:
The files are described in a file group element named `fileGrp`. The `fileGrp` element contains the descriptions of the files including their checksum and location. It is possible nest `fileGrp` elements inside of other `fileGrp` elements, however the CSIP such hierarchical nesting SHOULD NOT be used.